src/AdminList/CookieAdminListConfigurator.php

```diff
@@ -44,6 +61,12 @@ public function buildFilters()
     {
         $this->addFilter('name', new StringFilterType('name'), 'kuma.cookie.adminlists.cookie.name');
         $this->addFilter('type', new EnumerationFilterType('id', 't'), 'kuma.cookie.adminlists.cookie.type', $this->getCookieTypes());
+        if ($this->domainConfiguration->isMultiDomainHost()) {
+            $hosts = $this->domainConfiguration->getHosts();
+            $domains = array_combine($hosts, $hosts);
+            $domains = array_merge(array('' => 'kuma.cookie.adminlists.filter.all'), $domains);
+            $this->addFilter('domain', new ORM\EnumerationFilterType('domain'), 'kuma.cookie.adminlist.filter.domain', $domains);
```

Comment:
`EnumerationFilterType` is already imported, so use it directly and remove the ORM import in the uses